gems/pending/appliance_console/cli.rb

```diff
@@ -48,13 +52,25 @@ def database?
     end
 
     def local_database?
-      hostname && local?(hostname)
+      database? && local?(hostname)
     end
 
     def certs?
       options[:postgres_client_cert] || options[:postgres_server_cert] || options[:http_cert]
     end
 
+    def uninstall_ipa?
+      options[:uninstall_ipa]
```

Comment:
Won't this be boolean? If so, then "false" is a valid value, but you are educating if false is passed, which is weird.  Perhaps you want to use the `:has_uninstall_ipa` option instead to see if the value has been passed?